kratos/utilities/geometry_utilities.cpp

```diff
@@ -116,7 +116,7 @@ void GeometryUtils::EvaluateHistoricalVariableGradientAtGaussPoint(
 
 // template instantiations
 
-template void GeometryUtils::EvaluateHistoricalVariableValueAtGaussPoint<array_1d<double, 3>>(
```

Comment:
I am confused as to why you add this in the cpp

@roigcarlo you know the win stuff better, can you take a look please?

Reply:
It is to have noalias for non primitive types, and not to have noalias for doubles

Reply:
I get that but I never saw `KRATOS_API` in a source file

Reply:
I followed what is done in the earlier PR for stat app.. No idea why I did it :|, @roigcarlo could you maybe clarify where we need to put KRATOS_API to make it compile in windows?

Reply:
Explicit instabtiations without especialization.

To be completely honest still not sure if it is needed or its enough with the instantiation. If it passes thr ci withoit you can remove it.

Reply:
Could you may be add it to wiki :), how one should add KRATOS_API?. There are lots of developers who only has linux compilations in their workstations. So we rely on CI to capture windows compilation errors and it takes time also. :/

In this case, it was not even captured in original PR because, this was not used until now in #7039.

Reply:
I added a page here @roigcarlo @philbucher according to my experiences with trial and error. Could you please have a look?

https://github.com/KratosMultiphysics/Kratos/wiki/KRATOS_API-for-Windows-Compilation

Reply:
to me looks ok but I am not a Win expert :man_cartwheeling: